Gitlab/two-environment-pipeline-template/questions.json

```diff
@@ -78,8 +79,9 @@
     "isRequired": true
   }, {
     "key": "prod_stack_name",
-    "question": "What is the production stack name?",
-    "isRequired": true
+    "question": "Please specify a stack name for production environment",
+    "isRequired": true,
+    "default": "prod-stack"
```

Comment:
Upon second thought, should most likely use a fixed stack name. It's very reasonable for someone to enter a non-default testing environment name, at which point this suggestion will add confusion as there are now 2 terms for the same environment. Plus it's not very common to include the environment in stacks in general.